src/utils/ethereumUtils.ts

```diff
@@ -25,17 +28,29 @@ import {
   Linking,
   NativeModules,
 } from 'react-native';
+// @ts-expect-error ts-migrate(2305) FIXME: Module '"react-native-dotenv"' has no exported mem... Remove this comment to see the full error message
 import { ETHERSCAN_API_KEY } from 'react-native-dotenv';
 import { useSelector } from 'react-redux';
 import URL from 'url-parse';
+import {
+  DEFAULT_HD_PATH,
+  EthereumPrivateKey,
+  identifyWalletType,
+  WalletLibraryType,
+} from '../model/wallet';
+import {
+  EthereumAddress,
+  ParsedAddressAsset,
+  RainbowTransaction,
+} from '@rainbow-me/entities';
 import { getOnchainAssetBalance } from '@rainbow-me/handlers/assets';
 import {
   getProviderForNetwork,
   isTestnet,
   toHex,
 } from '@rainbow-me/handlers/web3';
 import isNativeStackAvailable from '@rainbow-me/helpers/isNativeStackAvailable';
-import networkTypes from '@rainbow-me/helpers/networkTypes';
+import networkTypes, { Network } from '@rainbow-me/helpers/networkTypes';
```

Comment:
you should be able to replace all instances of `networkTypes` with `Network` now